app/views/declaration/goods_location.scala.html

```diff
@@ -59,10 +60,10 @@
             sectionHeaderKey = None
         )
 
+        @goodsLocationExpander()
 
         @tariffExpander(GoodsLocationForm, request.declarationType)(messages, appConfig)
```

Comment:
Shouldn't `tariffExpander` be removed?

Reply:
No! Liz was very specific about that :smile: